python/ray/util/sgd/torch/torch_runner.py

```diff
@@ -263,7 +263,6 @@ def shutdown(self):
         """Attempts to shut down the worker."""
         del self.train_iterator
         del self.val_iterator
-        del self.training_operator
```

Comment:
This issue is really interesting... a few questions:

1. Any idea why this line in particular hangs?
2. Do we still need to delete `self.train_iterator` and `self.val_iterator` as well? Wondering if there is any additional context for why these lines were present in the first place.

As an alternative approach, should we just set these all to `None` instead?

Reply:
1. Don't know why in particular this hangs. But didn't want to spend too much time digging into this since we're going to end up replacing all this anyways.
2. We can; I don't think it matters either way. I'm not sure why this line was added in the first palce, but looks like there's no real reason for it (tests are all passing)